.gitignore

```diff
@@ -1,4 +1,7 @@
 # local development
 /config-local
 # Maven
-**/target/
+**target/
+**.settings
```

Comment:
From where did this entries come? Looks like your global GIT configuration does not include exclusions for your used IDE (looks like Eclipse). We don't want any IDE specific exclusions. If we start with them then we must respect any IDE.